tests/conftest.py

```diff
@@ -7,6 +7,6 @@
 def pytest_addoption(parser):
     """Define some command line options for integration and unit tests."""
     parser.addoption("--charm-file", action="store")
-    parser.addoption("--flask-app-image", action="store")
     parser.addoption("--test-flask-image", action="store")
     parser.addoption("--test-db-flask-image", action="store")
+    parser.addoption("--django-app-image", action="store")
```

Comment:
What is the reason for no longer needing flask and needing django please? Perhaps this was missed?

Reply:
The removed flask integration is originally used to test the default image shipped with the `flask-k8s` charm, which is not used anymore

Reply:
Does that mean the django app image is needed?

Reply:
I have remove the django image, thanks